A6httplib/A6httplib.cpp

```diff
@@ -27,8 +24,12 @@ A6httplib::~A6httplib()
 
 bool A6httplib::Post(String host, String path, String body)
 {
-     String X_IOTFY_ID = "24580020";
-     String X_IOTFY_CLIENT = "5dcfb9243fc64d3c1fd9de2a69c5bd";
+
+     // add your custom headers specific to IOT cloud service portal you are using.
+     // these are for Iotfy.co
+	 //  special Headers for making the POST request.
```

Comment:
yeah @skorokithakis I understand your point.  For this case, I am trying to implement a AddHeader function that may ease the work up and make it independent.